src/metabase/email/alert.hbs

```diff
@@ -10,7 +10,7 @@
                   <img class="icon" style="padding-top: 4px; padding-right: 16px; width: 100%; height: auto; max-width: 20px" src="cid:{{computed.icon_cid}}"/>
                 </td>
                 <td>
-                  <a class="title" href="{{computed.card_url}}" style="font-size: 20px; font-weight: 700; padding-bottom: 4px;">
+                  <a class="title" href="{{{computed.card_url}}}" style="font-size: 20px; font-weight: 700; padding-bottom: 4px;">
```

Comment:
Why was the old template OK with escaping?

Reply:
```clojure
(render-string "{{url}} | {{{url}}}" {:url "https://google.com?query=1&query=2"})
;; => "https://google.com?query&#x3D;1&amp;query&#x3D;2 | https://google.com?query=1&query=2"
(stencil.core/render-string "{{url}} | {{{url}}}" {:url "https://google.com?query=1&query=2"})
;; => "https://google.com?query=1&amp;query=2 | https://google.com?query=1&query=2"
```

handlebars is much more aggressive in escape, it's also escape `=`